.kokoro/tests/run_tests.sh

```diff
@@ -55,7 +55,7 @@ for file in **/pom.xml; do
     VERSION=$(grep -oP '(?<=<maven.compiler.target>).*?(?=</maven.compiler.target>)' pom.xml)
 
     # Check for changes to the current folder
-    if [ "$PARENT" -eq 0 ] && [ ",$JAVA_VERSIONS," = *",$VERSION,"* ]; then
+    if [ "$PARENT" -eq 0 ] && [[ $JAVA_VERSION = *"$VERSION"* ]]; then
```

Comment:
It's usually a good idea to escape variables w/ quotes in bash. 
```suggestion
    if [ "$PARENT" -eq 0 ] && [[ "$JAVA_VERSION" = *"$VERSION"* ]]; then
```